lib/wanda/operations/server.ex

```diff
@@ -19,10 +19,12 @@ defmodule Wanda.Operations.Server do
 
   require Logger
 
-  @default_timeout 5 * 60 * 1_000
-
   @impl true
-  def start_operation(operation_id, group_id, operation, targets, config \\ []) do
+  def start_operation(operation_id, group_id, operation, targets, config \\ [])
```

Comment:
This change is unrelated to the timeout feature.
But I already had it there, so...
It basically discards operations if coming `targets` list is empty